src/components/combo_box/combo_box.tsx

```diff
@@ -73,6 +74,16 @@ interface _EuiComboBoxProps<T>
   placeholder?: string;
   rowHeight?: number;
   singleSelection: boolean | EuiComboBoxSingleSelectionShape;
+  /**
+   * Creates an input group with element(s) coming before input. It only shows when the `singleSelection` is set to `true`.
```

Comment:
Again because singleSelection can be other than just a boolean
```suggestion
   * Creates an input group with element(s) coming before input. It won't show if `singleSelection` is set to `false`.
```